### ISSUE TITLE
Support for more languages

### ISSUE THREAD

**Issue by TheFrenchGhosty** [cloudrac3r/bibliogram#26]
For people that don't speak English it would be nice to have something for them so that they can have of a good user experience when using Bibliogram.

As someone who speak french, and translate other open source software in french (Invidious, FreeTube...) I would happily do it for Bibliogram.

**Comment by cloudrac3r** [comment]
Waiting on #28

**Comment by afriqs** [comment]
@cloudrac3r Not sure we have to wait for the settings page to begin to work on translations. Do you already have an idea on the files organization for various languages ? We could start it and for the moment use an application setting as default value :thinking:

**Comment by cloudrac3r** [comment]
That's a fair point. I'll think more about how translations would be implemented. Settings will come soon(tm) though since I am very aware of how many things are waiting on it.

**Comment by cloudrac3r** [comment]
We have the settings page, so at some point I'll work on stripping all the user-facing strings out of their pages and putting them all into a file that can be swapped out with translations.

**Comment by cloudrac3r** [comment]
Bibliogram is moving to sourcehut, so this GitHub thread should not be commented on in future. I won't actually lock the thread out of courtesy.

If anyone has further comments, please post them here instead: https://todo.sr.ht/~cadence/bibliogram-issues/9

**Comment by cloudrac3r** [comment]
@afriqs Hiya, it seems you were interested in translating? The functionality is now available. If you're interested, please leave a comment on sourcehut: https://todo.sr.ht/~cadence/bibliogram-issues/9. If you don't have an account on sourcehut, that's fine, you don't need one. You can leave a comment by sending an email. If you're not interested, you don't have to reply.
